apps/supply-chain/src/pages/home/create/address-form/AddressForm.tsx

```diff
@@ -0,0 +1,51 @@
+import { Hex } from '@gear-js/api';
+import { useApi } from '@gear-js/react-hooks';
+import { useForm } from '@mantine/form';
+import clsx from 'clsx';
+import { Input } from 'components';
+import { isValidHex } from 'utils';
+import commonStyles from '../Create.module.scss';
+import styles from './AddressForm.module.scss';
+
+type Props = {
+  id: string;
+  onSubmit: (value: { nftProgramId: Hex; ftProgramId: Hex }) => void;
+};
+
+const initialValues = {
```

Comment:
Move it in consts, helpers

Reply:
I prefer to not separate small encapsulated values

Reply:
Placeholder values are gone. 
As for utils, I prefer to not separate small encapsulated values